### ISSUE TITLE
Better pprint

### ISSUE THREAD

**Issue by MSeeker1340** [JuliaDiffEq/LabelledArrays.jl#32]
Although this is not a high priority it would be nice to be able to show the labels in `Base.show`, e.g.

```julia
julia> x = @LArray [1,2] (:a,:b);

julia> println(x)
[a=1, b=2]
```

To do this we need to look at https://github.com/JuliaLang/julia/blob/master/base/arrayshow.jl, which could be complicated.

**Comment by ChrisRackauckas** [comment]
If we print like that, it would be really nice to have that be reversible, i.e. solve https://github.com/JuliaDiffEq/LabelledArrays.jl/issues/12 so that it can easily copy-paste back into another LabelledArray.

**Comment by MSeeker1340** [comment]
Closed via #33.
